src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -67,22 +67,37 @@
       final Collection<Unit> transports) {
 
     final List<Unit> canBeTransported = sortByTransportCostDescending(units);
-    final List<Unit> canTransport = sortByTransportCapacityAscendingThenMovesDescending(transports);
+    final List<Unit> canTransport = sortByTransportCapacityDescendingThenMovesDescending(transports);
 
     // Add max units to each transport
     final Map<Unit, Unit> mapping = new HashMap<>();
+    Optional<Unit> finalTransport = Optional.empty();
     for (final Unit transport : canTransport) {
-      int capacity = TransportTracker.getAvailableCapacity(transport);
-      for (final Iterator<Unit> it = canBeTransported.iterator(); it.hasNext();) {
-        final Unit unit = it.next();
-        final int cost = UnitAttachment.get((unit).getType()).getTransportCost();
-        if (capacity >= cost) {
-          capacity -= cost;
-          mapping.put(unit, transport);
-          it.remove();
+
+      // Check if remaining units can all be loaded into 1 transport
```

Comment:
Add max units to each transport, means if we have 3 units, and 3 transports, we load 1 unit in each transport.

But "remaining units can all be loaded into 1 transport", means that with 3 units and 3 transports, we load the remaining 3 into just 1 if they will fit

So the comments are not consistent with how the code functions.